crates/typst/src/foundations/repr.rs

```diff
@@ -1,6 +1,7 @@
 //! Debug representation of values.
 
 use ecow::{eco_format, EcoString};
+use typst::utils::format::round_with_precision;
```

Comment:
This would probably be `crate::utils` instead (which would move this line under `use crate::foundations` below), though maybe we should just not import it at all and use the full path, since there's only one usage. Not sure what Laurenz thinks about this though.

Reply:
Yes, should be crate. Whether to import it or not, I don't have a strong preference. Could also import the module instead of the function and qualify just that below. I think that's relatively common for function in Rust in comparison to types.

Reply:
What about in download/watch files?

Reply:
There it's ok as-is.